rfcs/text/0000-authentication-fields.md

```diff
@@ -0,0 +1,136 @@
+# 0000: Authentication Fields RFC
+
+Stage: 0
+Date: 2021-08-08
+
+This RFC will provide normalization for fields related to authentication fields to assure that they are retained primarily in ECS core and in any extended fieldset when needed. These fields are important to normalize into ECS fields in order to maximize effectiveness of cross log utilization. 
+
+## Fields
+
+#### Log
+|Proposed Field Name|Type|Value|Description|
+| --- | --- | --- | --- |
+|authentication.scope|keyword|user_impersonation|azure.activitylogs.identity.claims.http://schemas_microsoft_com/identity/claims/scope|
+|authentication.requirement|keyword|multiFactorAuthentication|azure.activitylogs.properties.authenticationRequirement|
+|authentication.additional.details|keyword|MFA requirement satisfied by claim in the token|azure.activitylogs.properties.status.additionalDetails|
+|authentication.authorization|keyword|ROOTMANAGESHAREDACCESSKEY|azure.resource.authorization_rule|
+|authentication.isInteractive|keyword|FALSE|azure.signinlogs.properties.is_interactive|
+|authentication.risk|keyword|low|azure.signinlogs.properties.risk_level_aggregated|
+|authentication.id|Unique ID given across primary authentication and additional (multi factor) authentication.|||
+|authentication.protocol|Authentication Protocol (authproto)|||
+|authentication.token|Type of token for the login event|keyword| Unique token provided during the event|
+|authentication.token.identifier|Login token identifier/value|keyword| Unique identifier|
```

Comment:
Can these be assigned types, and then the descriptions moved over to that column?

Reply:
@kgeller  Thanks for the review. Change has been made